cdap-app-fabric/src/main/java/co/cask/cdap/internal/app/services/ServiceHttpServer.java

```diff
@@ -170,12 +170,22 @@ private NettyHttpService createNettyHttpService(Program program, String host,
                                       program.getApplicationId(),
                                       program.getName());
 
+    String versionId = program.getId().getVersion();
+    String versionedPathPrefix = String.format("%s/namespaces/%s/apps/%s/versions/%s/services/%s/methods",
```

Comment:
@gokulavasan  Is this still required for standlaone more? Can you add a TODO here with the JIRA to remove this when deterministic routing is implemented differently?